expo/tests/gbe/test_create_volunteer.py

```diff
@@ -78,3 +80,18 @@ def test_create_volunteer_no_post(self):
         login_as(ProfileFactory(), self)
         response = self.client.post(url)
         nt.assert_equal(response.status_code, 200)
+
+    def test_create_volunteer_post_with_submit_is_true(self):
+        url = reverse(self.view_name,
+                      urlconf='gbe.urls')
+        login_as(ProfileFactory(), self)
+        data = self.get_volunteer_form(submit=True)
+        response = self.client.post(url, data=data)
+        nt.assert_equal(response.status_code, 302)
```

Comment:
It would be better to add a "follow=True" clause to the client.post() call and then test the content of the response. Also, since it's a post we should expect it to attempt to affect the contents of the db, and we should test those changes. In this case, the test might be to get a count of Volunteers in the database (`Volunteer.objects.count()`) prior to the post, and compare that value to the same call after the post. Expect the value to increase by 1.